src/main/java/stringCalculator/User.java

```diff
@@ -0,0 +1,30 @@
+package stringCalculator;
+
+import java.io.ObjectStreamException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.StringTokenizer;
```

Comment:
불필요한 import문은 제거해주세요! 👀 
mac의 경우 `⌃ + ⌥ + O` 단축키를 통해서 사용하지 않는 import문을 제거할 수 있어요! 😉

```suggestion
import java.util.Arrays;
import java.util.List;
import java.util.Scanner;
```